src/main/reactjs/src/tests/cypress/support/page-objects/clerkTranslatorDetailsPage.ts

```diff
@@ -0,0 +1,102 @@
+import { AppRoutes } from 'enums/app';
+
+class ClerkTranslatorDetailsPage {
+  navigateById(id: number) {
+    cy.visit(
+      AppRoutes.ClerkTranslatorDetailsPage.replace(/:translatorId$/, `${id}`)
+    );
+  }
+
+  contentContainer() {
+    return cy.get('.clerk-translator-details-page__content-container');
+  }
+
+  backToRegisterBtn() {
+    return cy.get('.clerk-translator-details-page__back-btn');
+  }
+
+  editTranslatorInfoBtn() {
+    return cy.findByTestId(
+      'clerk-translator-details__contact-details__edit-btn'
+    );
+  }
+
+  addAuthorisationBtn() {
+    return cy.findByTestId(
+      'clerk-translator-details__authorisation-details__add-btn'
+    );
+  }
+
+  contactDetailsField(field: string) {
+    return cy
+      .findByTestId(`clerk-translator-details__contact-details__field-${field}`)
+      .find('div>input');
+  }
+
+  authorisationDetailsRow(id: number) {
+    return cy.findByTestId(`authorisations-table__id-${id}-row`);
```

Comment:
Noissa ei varmaan tarvita returnia?

Reply:
Tarvitaan kyllä, näiden paluuarvoja käytetään esim. `onClerkTranslatorDetailsPage.authorisationDetailsRow(3).should(..)`

Reply:
Aivan niin mutta tässä voisi sitten käyttää samaa mallia esim kuten clerkSendEmailPagessa on tehty eli noita voisi laittaa elementin alle?

Reply:
Joo, why not.

Reply:
Muutettu.